thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,48 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetPackages(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetPackages")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	manager := "pip"
+	ami := "ami-0e12ef25a5f7712a4"
+	opts := PackageFilterOptions{
+		Page:    0,
+		Limit:   10,
+		Manager: manager,
+	}
+	result, err := c.getPackages(ctx, ami, opts)
+	require.NoError(t, err)
+	assert.NotEmpty(result)
+	assert.Len(result, 10)
+	assert.Contains(result[0].Manager, manager)
+	name := "Automat"
+	opts = PackageFilterOptions{
+		Page:    0,
+		Limit:   5,
+		Name:    name,
+		Manager: manager,
+	}
+	result, err = c.getPackages(ctx, ami, opts)
+	require.NoError(t, err)
+	require.NotEmpty(t, result)
+	assert.Equal(result[0].Name, name)
+	assert.Contains(result[0].Manager, manager)
+	assert.Len(result, 1)
```

Comment:
this check should come before actually accessing a value in result, and should likely be a require so that 56 doesn't panic